### ISSUE TITLE
Absolutely Perfect. 😁👍

### ISSUE THREAD

**Issue by janborup** [Ginj-capture/Ginj#1]
**Hi @vicnevicne **

Today 14 July - My "reversed Ginj" rand out the the error "Jing's is not really sure what happened"... and it shutdown. 

Then I stumbled on this [[Ginj]], installed it for Windows and it just works like a charm. 😁

FANTASTIC good work... Thanks.

**Comment by vicnevicne** [comment]
Thanks for the kind words :-) . Yeah, they said Jing would continue working but I got the same error .

I just released Ginj v0.3.2 a few hours ago with a fix for Dropbox & Google Photos refresh token (silly mistake). Don't hesistate to update (you don't need to uninstall the previous version, it should get updated).

Starting from that version v0.3.2, auto-update should also work. I intend to add a few more features in the next few days and I'd be interested to know if you're getting automatic updates. Don't hesitate to tell me...

By the way, can you point one feature that should get the top priority in the [TODO list](https://github.com/Ginj-capture/Ginj/blob/master/todo.md) (or outside of it?)

**Comment by janborup** [comment]
Cheers @vicnevicne 

Yes... i also got the Dropbox/Google token expire, so super this is allready fixed.

I have spreed the good word about Ginj everywhere, and even try to create a Wikipedia infopage for Ginj, but they are a little hard to handle, but Ginj is allready on the Comparison sheet 👌

The only think i need on the ToDo list is "**Multi screen support**".. that worked perfect in Jing, but not in Ginj, so maybe you would consider this task.

/Cheers and Happy Coding.

PS: I'm only an C# programmer, and have no JAVA experience, so I'm not able to help you coding, but if you like to answer the community here on GitHub If I can do that.

**Comment by vicnevicne** [comment]
Yeah, I had also edited the Jing wikipedia page to mentionGinj as an alternative but the change got removed ;-)
I have seen your page about Ginj yesterday, thanks :-). I've edited it somehow and tried to rename it to just "Ginj" because I twant to avoir trademark issues (a reference in text is OK, but in the app name, or in the app itself, I'd rather not have any). As rename does not seem possible, would you mind if we just dropped that page and started another one with the name "Ginj" (or "Ginj (software)" if there is a duplicate) ? I'd be grateful if you can help in that regard. I also registered Ginj on AlternativeTo.net, but it's not validated yet. I've also answered a few topics on Techsmith's forums, but it seems impossible to find Ginj using for Google currently. Not that I'm targeting a success story - it's just a personal project written for fun - but I cannot apply for app validation at Dropbox until Ginj has been linked to Dropbox by 50 users :-( 
So in summary, if you can help with some PR stuff or SEO to reach a few more users (Windows currently, as early Linux feedback shows serious display issues and I guess Mac is similar), that would be greatly appreciated so I can focus on code :-).

**Comment by vicnevicne** [comment]
Oh, and multi-screen support, yes, indeed. 
I'm coding on a single large screen at home so I didn't think of it but I guess it would have been top on my TODO list if I installed it at work. Will add that. Thanks.

**Comment by vicnevicne** [comment]
Can you try starting Ginj and see if it auto-updates ? 
Hopefully you should get v0.3.3 which has multi-screen capture support ;-). See issue #2 
Otherwise you'll have to download and install it.

**Comment by janborup** [comment]
Thanks @vicnevicne 
The auto-update for my Win(x64) did not start nor asked to update, just by opening Ginj, so I downloaded it from the Release page, and that works fine.
Today I'm not on a Multi-Screen, so I will test this function in a couple of days.
Thanks again for the fine work.

**Comment by vicnevicne** [comment]
Thanks for the feedback. 
Auto-update is a feature of Install4J but it has several options and I admit I didn't take time to read it in detail, I just ticked the checkbox and hoped for the best... That obviously wasn't enough ;-)

**Comment by Creamers158** [comment]
Jing was the good, Ginj will be the best ;)

One thing though. When I added dropbox and wanted to share a photo I got this error in the log:
[ERROR] com.install4j.runtime.beans.actions.control.SetVariableAction [ID 278]: Execute action not successful after 2 ms
The upload was succesfull but it appeared that the emailadres was nog verified yet. Maybe adjust the error message for future users.     (Also a small typo:  Error creatiing shared link  -> Error creating shared link.)

Anyway, keep it going, love it already!!

**Comment by vicnevicne** [comment]
OK, I opened issue #7 to discuss this specific problem (oh, and the typo is fixed, thanks)

**Comment by janborup** [comment]
Hi @vicnevicne 

Yeaaaa... The updater works Smooth now.. 😊👍

![image](https://user-images.githubusercontent.com/6866443/88147064-fde40e00-cbfc-11ea-9186-a285079a6aa5.png)

Thanks

**Comment by vicnevicne** [comment]
Great. I'm tempted to close the issue, but I like its title so much that I will keep it open :-). 
If you have other comments, feel free to open dedicated issues though. Thanks !
